zebra/rtadv.c

```diff
@@ -1282,6 +1281,53 @@ DEFPY (no_ipv6_nd_ra_hop_limit,
 	return CMD_SUCCESS;
 }
 
+DEFPY (ipv6_nd_ra_retrans_interval,
+       ipv6_nd_ra_retrans_interval_cmd,
+       "ipv6 nd ra-retrans-interval (0-4294967295)$interval",
+       "Interface IPv6 config commands\n"
+       "Neighbor discovery\n"
+       "Advertisement Retransmit Interval\n"
+       "Advertisement Retransmit Interval in msec\n")
+{
+	VTY_DECLVAR_CONTEXT(interface, ifp);
+	struct zebra_if *zif = ifp->info;
+
+	if (if_is_loopback(ifp)
+	    || CHECK_FLAG(ifp->status, ZEBRA_INTERFACE_VRF_LOOPBACK)) {
+		vty_out(vty,
+			"Cannot configure IPv6 Router Advertisements on this interface\n");
```

Comment:
May be indicating the reason  also in the message on why we are disallowing this configuration is good. i.e. loopback interface.

Reply:
Agreed.  Changing it.